### PR TITLE
Prepackaged RedHat Model Servers

### LINKED ISSUE #1734: RedHat version of core servers
Provide RedHat certified images for:
  * sklearnserver
  * mlflow server
  * xgboost server
  * tfserving

### LINKED ISSUE #1776: Add explainer image version to configmap
Presently hardwired in Operator.

### PR DESCRIPTION
**Pull request by cliveseldon**
Fixes #1734 
Fixes #1776 

 * Adds a python base ubi s2i 
 * Updates all prepackaged model servers to be ubi based images
 * Updates all prepackaged model servers to use core version
 * Adds images for explainer-wrapper and seldon-storage-initializer
 * Adds redhat images for other servers
 * adds explainer image to configMap and allows related image override

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **cliveseldon**

If they are not already assigned, you can assign the PR to them by writing `/assign @cliveseldon` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Thu Apr 30 18:25:45 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=2

**Comment by seldondev** [comment]
Thu Apr 30 18:25:46 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=1

**Comment by seldondev** [comment]
Fri May  1 15:37:57 UTC 2020
The logs for [pr-build] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=3

**Comment by seldondev** [comment]
Fri May  1 15:37:59 UTC 2020
The logs for [lint] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/4.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=4

**Comment by seldondev** [comment]
Fri May  1 16:32:22 UTC 2020
The logs for [pr-build] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/5.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=5

**Comment by seldondev** [comment]
Fri May  1 16:33:40 UTC 2020
The logs for [lint] [6] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/6.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=6

**Comment by seldondev** [comment]
Sat May  2 16:25:07 UTC 2020
The logs for [pr-build] [7] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/7.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=7

**Comment by seldondev** [comment]
Sat May  2 16:25:11 UTC 2020
The logs for [lint] [8] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/8.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=8

**Comment by seldondev** [comment]
Sat May  2 17:42:00 UTC 2020
The logs for [lint] [10] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/10.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=10

**Comment by seldondev** [comment]
Sat May  2 17:43:18 UTC 2020
The logs for [pr-build] [9] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/9.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=9

**Comment by seldondev** [comment]
Sat May  2 18:24:24 UTC 2020
The logs for [lint] [12] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/12.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=12

**Comment by seldondev** [comment]
Sat May  2 18:24:25 UTC 2020
The logs for [pr-build] [11] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/11.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=11

**Comment by seldondev** [comment]
Thu May  7 08:25:04 UTC 2020
The logs for [lint] [14] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/14.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=14

**Comment by seldondev** [comment]
Thu May  7 08:25:20 UTC 2020
The logs for [pr-build] [13] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/13.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=13

**Comment by seldondev** [comment]
Thu May  7 18:02:58 UTC 2020
The logs for [lint] [16] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/16.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=16

**Comment by seldondev** [comment]
Thu May  7 18:04:16 UTC 2020
The logs for [pr-build] [15] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/15.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=15

**Comment by seldondev** [comment]
@cliveseldon: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 474e06ffdfc6dbce536820d00e1713c7455cb8a4 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Thu May 14 08:29:41 UTC 2020
The logs for [lint] [18] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/18.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=18

**Comment by seldondev** [comment]
Thu May 14 08:29:42 UTC 2020
The logs for [pr-build] [17] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/17.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=17

**Comment by seldondev** [comment]
Fri May 15 17:55:43 UTC 2020
The logs for [lint] [20] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/20.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=20

**Comment by seldondev** [comment]
Fri May 15 17:55:48 UTC 2020
The logs for [pr-build] [19] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/19.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=19

**Comment by seldondev** [comment]
Fri May 15 18:12:35 UTC 2020
The logs for [integration] [21] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/21.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=21

**Comment by seldondev** [comment]
@cliveseldon: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Mon May 18 07:26:07 UTC 2020
The logs for [integration] [23] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/23.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=23

**Comment by seldondev** [comment]
@cliveseldon: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@cliveseldon: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test integration`
pr-build | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@cliveseldon: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test this`
integration | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@cliveseldon: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test this`
integration | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test integration`
notebooks | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@cliveseldon: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test this`
integration | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test integration`
notebooks | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test notebooks`
lint | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test lint`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: components/alibi-detect-server/Dockerfile [hunk 1]
```diff
@@ -1,14 +1,15 @@
-FROM python:3.7
-
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends git \
-    && apt-get purge -y --auto-remove \
-    && rm -rf /var/lib/apt/lists/*
-
-WORKDIR /workspace
+FROM registry.access.redhat.com/ubi8/python-36
```
[the remark this answers is not in the document]
Comment (cliveseldon) [reply components/alibi-detect-server/Dockerfile:1]: There doesn't seem to be a ubi 37 image available when I did the updates. So suggest we update this when it becomes available in a later PR?

**Comment by seldondev** [comment]
Sat May 23 16:48:25 UTC 2020
The logs for [pr-build] [28] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/28.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=28

**Comment by seldondev** [comment]
Sat May 23 16:49:49 UTC 2020
The logs for [lint] [29] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/29.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=29

**Comment by seldondev** [comment]
Sat May 23 17:06:43 UTC 2020
The logs for [integration] [30] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/30.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=30

**Comment by seldondev** [comment]
@cliveseldon: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test notebooks`
integration | 265818a6a819bf99e56b083e14ed52a465af10c8 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Sat May 23 17:23:55 UTC 2020
The logs for [integration] [31] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/31.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=31

**Comment by seldondev** [comment]
@cliveseldon: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test notebooks`
integration | 265818a6a819bf99e56b083e14ed52a465af10c8 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Sun May 24 08:07:02 UTC 2020
The logs for [integration] [32] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/32.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=32

**Comment by seldondev** [comment]
@cliveseldon: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test notebooks`
integration | 265818a6a819bf99e56b083e14ed52a465af10c8 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Sun May 24 09:02:13 UTC 2020
The logs for [integration] [33] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/33.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=33

**Comment by seldondev** [comment]
@cliveseldon: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test notebooks`
integration | 265818a6a819bf99e56b083e14ed52a465af10c8 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Sun May 24 09:51:50 UTC 2020
The logs for [pr-build] [34] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/34.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=34

**Comment by seldondev** [comment]
Sun May 24 09:51:51 UTC 2020
The logs for [lint] [35] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/35.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=35

**Comment by seldondev** [comment]
Sun May 24 09:52:55 UTC 2020
The logs for [lint] [37] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/37.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=37

**Comment by seldondev** [comment]
Sun May 24 09:54:11 UTC 2020
The logs for [pr-build] [36] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/36.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=36

**Comment by seldondev** [comment]
Sun May 24 09:58:06 UTC 2020
The logs for [lint] [39] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/39.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=39

**Comment by seldondev** [comment]
Sun May 24 09:58:20 UTC 2020
The logs for [pr-build] [38] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/38.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=38

**Comment by seldondev** [comment]
Sun May 24 10:18:34 UTC 2020
The logs for [integration] [40] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/40.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=40

**Comment by seldondev** [comment]
@cliveseldon: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 48f57e471b1705a2f62fb65683964da0deb975c5 | [link]() | `/test notebooks`
integration | 6a84725592e492c1ba34e1b03b23c6bf49399895 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Sun May 24 14:24:51 UTC 2020
The logs for [pr-build] [41] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/41.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=41

**Comment by seldondev** [comment]
Sun May 24 14:24:59 UTC 2020
The logs for [lint] [42] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/42.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=42

**Comment by seldondev** [comment]
Sun May 24 14:45:29 UTC 2020
The logs for [integration] [43] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/43.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=43

**Comment by seldondev** [comment]
Sun May 24 18:46:42 UTC 2020
The logs for [notebooks] [44] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1775/44.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1775 --build=44

**Comment by seldondev** [comment]
@cliveseldon: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | b17912fd03560223b1ccec189dcf324841741998 | [link]() | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by axsaucedo** [comment]
Last few commits after comment look good 👍 
/approve

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/SeldonIO/seldon-core/pull/1775#issuecomment-633417410" title="Approved">axsaucedo</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [axsaucedo]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**